src/plugins/data_views/common/data_views/data_views.ts

```diff
@@ -261,7 +265,10 @@ export interface DataViewsServicePublicMethods {
    * @params savedObject - Data view saved object
    * @params displayErrors - If set false, API consumer is responsible for displaying and handling errors.
    */
-  savedObjectToSpec: (savedObject: SavedObject<DataViewAttributes>) => DataViewSpec;
+  savedObjectToSpec: (
```

Comment:
I'm not sure why this is needed as `savedObjectToSpec` doesn't display toasts.

Reply:
@mattkime Right, I cleaned that up.